hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This release adds support for the Array API's `2023.12 release
+<https://data-apis.org/array-api/2023.12/>`_ via the ``api_version`` argument in
```

Comment:
Link 404's; I presume we should hold the corresponding Hypothesis release until https://data-apis.org/array-api/latest/ points to 2023.12?

Reply:
Whole docs were down infact yesterday, I might of accidentally deleted a certain `gh-pages` branch yesterday :sweat_smile: Fixed now!